.template/fuel-toolchain.toml

```diff
@@ -0,0 +1,6 @@
+[toolchain]
+channel = "latest-2023-01-17"
```

Comment:
```suggestion
channel = "latest-2023-01-18"
```

This should be [18th instead](https://github.com/FuelLabs/fuelup/tree/gh-pages/channels/latest). There should probably be a feature to easily list the dates you can declare in fuelup though

Reply:
Nice catch, missed that one. I strongly agree with the necessity of such a feature

Reply:
https://github.com/FuelLabs/fuelup/pull/375

PR up for such a feature